postcodes.ts

```diff
@@ -1,24 +1,24 @@
 const base_endpoint = "https://api.postcodes.io/postcodes/";
 
 // Quicker logging helper function
-export function log(msg) {
+export function log(msg: any) {
```

Comment:
Would this always just be a `string`?
Perhaps later look at if you switched that over - what errors does typescript throw?

any can be helpful for getting you underway but also sidesteps the type checking we're aiming for here.
(granted this is just a quick error function!)

Reply:
`log()` is basically a helper function I make for myself to avoid having to type out console.log every time...

I thought it was fine to leave this as `:any` because I wouldn't always want to log a string while debugging (could be an object, could be an int, etc).